### ISSUE TITLE
Start and restart individual tasks in turbo run

### ISSUE THREAD

**Issue by cmaycumber** [vercel/turbo#2946]
### Which project is this feature idea for?

Turborepo

### Describe the feature you'd like to request

Currently I've been using the library: `https://www.npmjs.com/package/run-pty` to start all tasks in parallel but also allows restarting individual tasks, and providing input to individual tasks via the console. 

This would be helpful because it allows developers to run a large amount of tasks in parallel but if one fails or stalls and can be restarted without having to restart all the packages in the monorepo.  

### Describe the solution you'd like

I'd like `turbo run` to have the ability to restart individual tasks and accept standard input for individual tasks. 

### Describe alternatives you've considered

Using `run-pty` instead of `turbo run` for development.

**Comment by mehulkar** [comment]
Closing in favor of https://github.com/vercel/turbo/issues/986 and https://github.com/vercel/turbo/issues/1235

**Comment by valeriangalliat** [comment]
Those 2 issues don't seem to be related? Watch mode and forward stdin are nice to have but it doesn't seem that both of those would allow starting and restarting individual tasks in turbo run

**Comment by mehulkar** [comment]
hey @valeriangalliat I'll re-open this, because this issue is larger in scope compared to Watch Mode and forwarding stdin, but wanted to explain my thinking. The original issue description 

> allows restarting individual tasks

Watch Mode is about re-running individual tasks when a related file or env var changes. I felt like that's one of the primary reasons to "restart" a task, but you're right that it's not exactly the same as a persistent server that needs to be restarted.

> providing input to individual tasks via the console

This is about forwarding stdin, but `run-pty` ismore extensive (with a dashboard UI to select which task gets stdin), etc. I imagine that that kind of interface will be a second interation of forwarding stidn! 

In any case, I'll re-open this for now!
